test-app-async/src/main/resources/application-localstack.yml

```diff
@@ -11,6 +11,11 @@ hmpps.sqs:
       dlqName: async-outbound-dlq
       subscribeTopicId: outboundtopic
       subscribeFilter: '{"eventType":[ "offender.movement.reception"] }'
+    asyncqueue:
+      dlqName: async-async-dlq
+      queueName: async-async-queue
+      asyncDlqClient: true
+      asyncQueueClient: true
```

Comment:
The async client for running the test app locally. The strange name is to keep with existing naming convention - will probably change to `reactive-async-dlq` when I do some refactoring.